Client/mods/deathmatch/logic/luadefs/CLuaElementDefs.cpp

```diff
@@ -2062,6 +2123,38 @@ int CLuaElementDefs::SetElementVelocity ( lua_State* luaVM )
     return 1;
 }
 
+int CLuaElementDefs::SetElementTurnVelocity ( lua_State* luaVM )
+{
+    CClientEntity* pEntity;
+    CVector vecTurnVelocity;
+
+    CScriptArgReader argStream ( luaVM );
+    argStream.ReadUserData ( pEntity );
+    argStream.ReadVector3D ( vecTurnVelocity );
+
+    // Verify the arguments
+    if ( !argStream.HasErrors () )
+    {
+        if ( pEntity )
```

Comment:
This needs changing too (see [first comment](https://github.com/multitheftauto/mtasa-blue/pull/73/files#r78756626))